docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md

```diff
@@ -170,7 +170,7 @@ Currently, you can only edit Snowflake Data Source settings.
 
     - **Account identifier**: Enter new Snowflake account or locator information. The locator value must include the geographical region. For example, `us-east-1`. To locate these values see [Account Identifiers](https://docs.snowflake.com/en/user-guide/admin-account-identifier).
 
-    - **Password/environment variable**: Enter a Snowflake password or `${GX_CLOUD_SNOWFLAKE_PASSWORD}`. If you haven't set this variable, see [Set up GX Cloud](../set_up_gx_cloud.md).
+    - **Password**: Enter your Snowflake password.
```

Comment:
I think we should re-iterate that we recommend using a service account in these steps (password and username); I skimmed the prereqs section and missed the "service account" bit.

Reply:
@DrewHoo, I have updated the content as recommended.